tests/integration/exec.bats

```diff
@@ -57,3 +57,42 @@ function teardown() {
   [[ ${lines[1]} == *"."* ]]
   [[ ${lines[2]} == *".."* ]]
 }
+
+@test "runc exec ls -la with --cwd" {
+  # run busybox detached
+  runc run -d --console /dev/pts/ptmx test_busybox
+  [ "$status" -eq 0 ]
+
+  wait_for_container 15 1 test_busybox
+
+  runc exec --cwd /bin test_busybox pwd
+  [ "$status" -eq 0 ]
+
+  [[ ${output} == *"/bin"* ]]
```

Comment:
It's not a bats issue. For some reason, runc acts weirdly with `pwd`. Here's an strace log for runC just doing `runc exec test pwd`.

```
[pid 22107] getcwd("/", 192)            = 2
[pid 22107] fstat(1, {st_mode=S_IFCHR|0600, st_rdev=makedev(136, 7), ...}) = 0
[pid 22107] mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7ffbaf0b6000
[pid 22107] write(1, "/\n", 2)          = 2
[pid 22099] <... read resumed> "/\r\n", 32768) = 3
[pid 22107] exit_group(0)               = ?
[pid 22099] futex(0xc589d0, FUTEX_WAKE, 1) = 1
[pid 22098] <... futex resumed> )       = 0
[pid 22099] write(1, "/\r\n", 3/
 <unfinished ...>
[pid 22098] select(0, NULL, NULL, NULL, {0, 20} <unfinished ...>
[pid 22099] <... write resumed> )       = 3
```